src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -8,21 +8,27 @@ description: "Learn about the Azure Active Directory Single Sign-On (SSO)."
 
 1. Log in to the [Azure portal](https://portal.azure.com/).
 
-2. (If you don't require a permission group for Sentry, skip to Step 4.) Under Manage Azure Active Directory (the shield picture), click "View", which takes you to the Organization Overview page. Now, proceed to Step 3.
+1. Under "Manage Azure Active Directory" (the shield picture), click "View", which takes you to the Organization Overview page.
 
-3. In the search bar, search for "Groups" and navigate to "Groups." From there, create a new group, add an owner, and members to the group. Set "Group type" to Office 365. For more details about group creation, see the [Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+1. If you don't require a permission group for Sentry, skip to Step 5.
 
-4. Return to the Overview page. In the search bar, search for "Enterprise Applications" and navigate to "Enterprise Applications." Click "+ New application". Then, select "Non-gallery application" and create a non-gallery application for Sentry. Once created, you'll be directed to "Sentry - Overview.
+1. In the search bar, search for "Groups" and navigate to it. From there, create a new group, add an owner, and members to the group. Set "Group type" to Office 365. For more details about group creation, see the [Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+1. Return to the Overview page. In the search bar, search for "Enterprise Applications" and navigate to it. Click `+ New application`. Then, search for "Sentry" and create the application for Sentry. 
```

Comment:
This is the main change. We have a Sentry app in the AzureAD gallery, so we should use that instead of asking the user to select `Non-gallery application`.